### PR TITLE
RHIROS-1315 process only RHEL systems

### PR DESCRIPTION
**Pull request by r14chandra**
## Why do we need this change? :thought_balloon:

[RHIROS-1315](https://issues.redhat.com/browse/RHIROS-1315)

## Documentation update? :memo:

- [ ] Yes
- [X] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [X] Bugfix
- [ ] New Feature
- [ ] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added
- [ ] DB Migration Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

File: ros/lib/constants.py [hunk 1]
```diff
@@ -27,3 +27,8 @@ class SubStates(Enum):
 class CloudProvider(Enum):
     """Supported cloud providers"""
     AWS = 'aws'
+
+
+class OperatingSystem(Enum):
```
Comment (upadhyeammit) [inline ros/lib/constants.py:32]: It does not seem we have plan to support other Operating Systems? so do we need Enum class? I absolutely understand keeping the implementation similar to other constants, however having question as well?

Comment (r14chandra) [reply ros/lib/constants.py:32]: BUs are still discussing. So, it is fine to keep Enum.
You can refer to [this](https://redhat-internal.slack.com/archives/C01C1GVT3TP/p1693230168469569?thread_ts=1693223068.533649&cid=C01C1GVT3TP) slack thread.

File: ros/lib/utils.py [hunk 2]
```diff
@@ -307,11 +314,14 @@ def highlights_instance_types(queryset, highlight_type):
...
     cloud_provider = ''
+    operating_system = ''
     if reporter == 'INSIGHTS ENGINE':
         is_ros = msg["input"]["platform_metadata"].get("is_ros")
-        cloud_provider = msg["results"]["system"]["metadata"].get('cloud_provider')
+        cloud_provider = msg["results"]["system"]["metadata"].get("cloud_provider")
+        operating_system = msg["input"]["host"]["system_profile"].get("operating_system")
     elif reporter == 'INVENTORY EVENTS':
         cloud_provider = msg['host']['system_profile'].get('cloud_provider')
+        operating_system = msg['host']['system_profile'].get('operating_system')
```
Comment (upadhyeammit) [inline ros/lib/utils.py:324]: Nit: Can we use either single or double quotes?

**Review by upadhyeammit — APPROVED** [review]
This looks good to me! Added a nitpick

**Pull request merged**